images/installer/Dockerfile.upi.ci

```
@@ -63,7 +63,8 @@ RUN yum update -y && \
       python3-pyyaml \
       bind-utils \
       util-linux \
-      xz && \
+      xz \
+      qemu-img.x86_64 && \
```

If we can not hardcode the arch, it's better
```suggestion
      qemu-img && \
```